doc/userguide/support-status.rst

```diff
@@ -0,0 +1,490 @@
+--------------------
+
+Tier 1
+~~~~~~
+
+.. table::
+   :widths: 15 15 30 40
+   :width: 100%
+
+   +-------------+-------------+-------------+-------------+
+   |Architecture |Support      |QA           |Notes        |
+   +=============+=============+=============+=============+
+   |x86_64       |Core team    |             |             |
+   +-------------+-------------+-------------+-------------+
+   |ARM8-64bit   |Core team    |             |             |
+   +-------------+-------------+-------------+-------------+
+
+Tier 2
+~~~~~~
+
+.. table::
+   :widths: 15 15 30 40
+   :width: 100%
+
+   +-------------+-------------+-------------+-------------+
+   |Architecture |Support      |QA           |Notes        |
+   +=============+=============+=============+=============+
+   |ARM7-32bit   |Core team    |             |             |
+   +-------------+-------------+-------------+-------------+
+   |i386         |Core team    |             |             |
+   +-------------+-------------+-------------+-------------+
+
+Community
+~~~~~~~~~
+
+.. table::
+   :widths: 15 15 30 40
+   :width: 100%
+
+   +-------------+-------------+---------------------------+---------------------------------------------+
+   |Architecture |Support      |QA                         |Notes                                        |
+   +=============+=============+===========================+=============================================+
+   |PPC64el      |             |Part of Fedora automated QA|Access can be arranged through IBM dev cloud |
+   +-------------+-------------+---------------------------+---------------------------------------------+
+   |PPC64        |Victor Julien|                           |No access except an old Mac G5 Victor has    |
+   +-------------+-------------+---------------------------+---------------------------------------------+
+   |PPC32        |Victor Julien|                           |No access except an old Mac G4 Victor has    |
+   +-------------+-------------+---------------------------+---------------------------------------------+
+
+High Level Features
+-------------------
+
+Capture support
+~~~~~~~~~~~~~~~
+
+Tier 1
+^^^^^^
+
+.. table::
+   :width: 100%
+
+   +----------------+-------------------------+----+-----------------------------+
+   | Capture Type   | Maintainer              | QA | Notes                       |
+   +================+=========================+====+=============================+
+   |AF_PACKET       |Core team / Eric Leblond |    |Used by Security Onion, SELKS|
+   +----------------+-------------------------+----+-----------------------------+
+   |NETMAP (FreeBSD)|Core team / Victor Julien|    |Used by OPNsense             |
+   |                |+ Aleksey Katargin       |    |                             |
+   +----------------+-------------------------+----+-----------------------------+
+   |NFQUEUE         |Core team / Andreas Herz |    |                             |
```

Comment:
core team

Reply:
we should remove all personal names maybe?